### PR TITLE
[#187474483] Decrypt SES Credentials via KMS

### PR DESCRIPTION
**Pull request by eifelmicha**

### CODE REVIEW

**Comment by eifelmicha** [comment]
@aberresch 
equesting review
requesting functional acceptance
requesting merge approval

File: tasks/main.yml [hunk 1]
```diff
@@ -1,12 +1,21 @@
 # vim:ft=ansible:ts=2:et:
 ---
+- name: Decrypt SES Credentials via KMS inside AWS
+  when: ansible_system_vendor == 'Amazon EC2'
+  # when AWS Condition does not work for Lints..
+  block:
+    - name: Decypt SES Credentials via KMS # noqa jinja[invalid]
```
Comment (aberresch) [inline tasks/main.yml:7]: Decrypt

**Review by aberresch — CHANGES_REQUESTED** [review]
Kleinigkeit

**Comment by eifelmicha** [comment]
@aberresch
equesting review
requesting functional acceptance
requesting merge approval

**Comment by aberresch** [comment]
review: lgtm
functionally accepted
merge approved

**Review by aberresch — APPROVED** [review]

**Pull request merged**